src/lib/helpers/GetKey.js

```diff
@@ -0,0 +1,3 @@
+export const getKey = (prefix, currentKey, parentKeyPath, marginLeft) => {
```

Comment:
just put all the helpers in one file, probably call it util.js or helpers.js

Reply:
resolved by moving all the helpers to a util.js file